awx/sso/middleware.py

```diff
@@ -20,9 +18,12 @@
 
 class SocialAuthMiddleware(SocialAuthExceptionMiddleware):
 
-    def process_view(self, request, callback, callback_args, callback_kwargs):
-        if request.path.startswith('/sso/login/'):
-            request.session['social_auth_last_backend'] = callback_kwargs['backend']
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        response = self.process_request(request)
```

Comment:
I just noticed that this middleware has a non-trivial middlware parent class.  You may need to confirm the presence or absence of a `.process_response()` method and adjust accordingly here.

Reply:
No `.process_response()` method found here (in parent class): https://github.com/python-social-auth/social-app-django/blob/master/social_django/middleware.py